lib/compat/wordpress-6.1/theme.json

```diff
@@ -215,6 +215,20 @@
 								"margin-left": "auto !important",
 								"margin-right": "auto !important"
 							}
+						},
+						{
+							"selector": ":where(.has-global-content-size) > :where(:not(.alignleft):not(.alignright):not(.alignfull))",
+							"rules": {
+								"max-width": "var(--wp--style--global--content-size)",
+								"margin-left": "auto !important",
+								"margin-right": "auto !important"
+							}
+						},
+						{
+							"selector": ":where(.has-global-wide-size) > .alignwide",
+							"rules": {
+								"max-width": "var(--wp--style--global--wide-size)"
+							}
```

Comment:
I'm wondering if we can use the same classname for these rules, since they have different child selectors.

Would it work to have a single `has-global-content-sizes` classname, so the second rule would become: `:where(.has-global-content-sizes) > .alignwide`?

Reply:
Yeah, we should be able to do that. Relatedly, I'm thinking that if there's only one of these sizes set, we should set the other to the same value. We already use this logic [in `layout.php`](https://github.com/WordPress/gutenberg/blob/trunk/lib/block-supports/layout.php#L49) so makes sense to use it here too.

Reply:
Hmm, can't seem to get fallback values to work for the variables defined in `theme.json`. I wonder if we're disallowing commas in the values or something?

Reply:
I suspect `safecss_filter_attr` has a problem with the commas, [although its docs](https://developer.wordpress.org/reference/functions/safecss_filter_attr/) say it supports `var`. I'll have a look on Trac and see if there's any ticket about that.

In the meantime, I made the fallback work by always setting values for both variables to whichever value exists.

Reply:
Nice, always setting both values looks like a good way to handle the fallback for now!